mlos_bench/mlos_bench/storage/sql/storage.py

```diff
@@ -27,14 +27,26 @@ class SqlStorage(Storage):
 
     def __init__(self, tunables: TunableGroups, service: Optional[Service], config: dict):
         super().__init__(tunables, service, config)
-        log_sql = self._config.pop("log_sql", False)
-        url = URL.create(**self._config)
-        self._repr = f"{url.get_backend_name()}:{url.database}"
+        lazy_schema_create = self._config.pop("lazy_schema_create", False)
```

Comment:
Useful for tests to avoid trying to connect to a database during unit test loading of the configs.